index.html

```diff
@@ -169,344 +169,6 @@ <h3>Sponsors & Partners</h3>
     </div>
   </section>
 
-  <!-- Rules, Submissions & FAQ -->
-  <section id="rules-submission-faq" class="wrapper style2 fade-up">
-    <div class="inner">
-      <h2>Rules, Submissions &amp; FAQ</h2>
```

Comment:
So likely underneath here, a quick blurb to say that all this information has moved to `(add hyperlink)` a separate page.

Reply:
Let's include this in the above story, as I'd like to see how Marjorie and everyone else want to re-organize the page.

To me people already understand that links either take them to a new part of a page, or a new page entirely so I don't know if it's necessary to add an explainer.

I agree we should sort it out though, I just want to have some more context and input.

Reply:
Never mind, disregard since scrolling effect works.  TIL relative links are A-OK for this template.